### PR TITLE
Add help for package authors

### PR DESCRIPTION
**Pull request by MikeInnes**
Making it clear what we require for a merge, and that feedback on names etc is optional.

### CODE REVIEW

File: AttoBot/lambda_function.py [hunk 1]
```diff
@@ -25,6 +25,12 @@
 
 SECRET = os.environ["SECRET"]
 
+TAG_REQ = "\n".join((
+    "Please make sure that:",
+    "- [ ] CI passes for supported Julia versions (if applicable).",
+    "- [ ] Version bounds are up to date."
```
Comment (tkelman) [inline AttoBot/lambda_function.py:31]: this should probably be elaborated on a bit

File: AttoBot/lambda_function.py [hunk 2]
```diff
@@ -25,6 +25,12 @@
 
 SECRET = os.environ["SECRET"]
 
+TAG_REQ = "\n".join((
+    "Please make sure that:",
+    "- [ ] CI passes for supported Julia versions (if applicable).",
```
Comment (tkelman) [inline AttoBot/lambda_function.py:30]: worth mentioning now that JuliaCIBot will run tests against the package and its reverse dependencies to see if breaking changes are going to cause problems

File: AttoBot/lambda_function.py (hunk 1 above)
Comment (MikeInnes) [reply AttoBot/lambda_function.py:31]: Yeah, this won't be totally clear to a newbie. Do you have anything we can link to?

File: AttoBot/lambda_function.py (hunk 2 above)
Comment (MikeInnes) [reply AttoBot/lambda_function.py:30]: Looks like this isn't stable yet, right? Once it is we can hopefully just rely on that entirely for package tests.

Comment (tkelman) [reply AttoBot/lambda_function.py:30]: It's getting there pretty quickly. We should ask @nkottary to add a pending status indicator when the job gets triggered so we don't overlook it.

Both are useful to look at. JuliaCIBot isn't going to do well with packages that need custom setup, depend on non-default programs or libraries, etc. Those are usually easier to set up by a package author in their own .travis.yml and are likely non-trivial to replicate completely generally.

File: AttoBot/lambda_function.py (hunk 1 above)
Comment (tkelman) [reply AttoBot/lambda_function.py:31]: Not in one place yet, but I should really write up a package doc section on the subject.

Comment (MikeInnes) [reply AttoBot/lambda_function.py:31]: That would be a huge boon as far as distributing the workload is concerned, I think. I suspect even many package authors aren't sure exactly what they should be doing here.

File: AttoBot/lambda_function.py (hunk 2 above)
Comment (nkottary) [reply AttoBot/lambda_function.py:30]: Yes, I can add the pending status indicator for the package tests. Also, I'm going to show the reverse dependency results separately as a comment as that could take an arbitrary amount of time to complete.

Comment (tkelman) [reply AttoBot/lambda_function.py:30]: can we parallelize those across multiple workers?

Comment (nkottary) [reply AttoBot/lambda_function.py:30]: It is already running in parallel on a cluster but we currently don't have dynamic scaling capabillity.

File: AttoBot/lambda_function.py (hunk 1 above)
Comment (tkelman) [reply AttoBot/lambda_function.py:31]: I'll put that on my to-do list, hopefully right after rc2

File: AttoBot/lambda_function.py (hunk 2 above)
Comment (tkelman) [reply AttoBot/lambda_function.py:30]: the pending status is working nicely, thanks!

**Comment by simonbyrne** [comment]
I like this. Would you like me to merge it now?

**Comment by MikeInnes** [comment]
Go for it, we can tweak as needed.

**Pull request merged**